src/assets/scss/components/_searchpage.scss

```diff
@@ -0,0 +1,175 @@
+@use '../abstracts/' as *;
+.search-page {
+  .fill {
+    min-height: 100vh;
+    height: 100%;
+  }
+  h1 {
+    // set color to white
+    color: $white;
+    font-size: 1.5rem;
+    margin-bottom: 1.5rem;
```

Comment:
You can try to use h1, h2, h3, ..., h6 to conform your design.
Or   <element className="text-white"> of bootstrap  instead.
But every think looking good